### ISSUE TITLE
Improve Meteor installation Process

### ISSUE THREAD

**Issue by harryadel** [meteor/meteor-feature-requests#433]
Inspired by https://github.com/meteor/meteor-feature-requests/issues/390, which aims to improve the Meteor installation experience, this [article](https://medium.com/@kschingiz/weirdness-of-meteorjs-framework-ef4373ff487b) mentions a fundamental problem in the Meteor installation process:

>Meteor requires 345mb database on your machine
Have you ever wondered why installing meteor takes a long time? Well, here is the answer: Meteor downloads 345mb database. You may never need the information stored in the database, but Meteor does not care.

>Don’t believe me? Just open ~/.meteor/package-metadata/v2.0.1 folder

Good internet is yet to be a standard in many parts of the world, so fixing this would open up Meteor to a greater community and further lessen the number of obstacles to adopting Meteor.

Generally speaking, the article above touches upon many things which Meteor can improve, but for the sake of brevity, I've kept this issue only for the first problem.

**Comment by StorytellerCZ** [comment]
I agree that it is not optimal and I'm open to suggestions on how to "fix" this. Right now I have other priorities and I'm eyeing #390 to look into probably around Meteor 2.4 or 2.5 if time permits.

**Comment by santiagopuentep** [comment]
Yes, please improve the CLI, especially when installing, updating and deploying. It's horrible not only because it's soooo slow, but because it doesn't give you feedback on what it's doing and the deploying sometimes hangs without letting the user know if it's actually frozen or if it's just moving slowly.

With new tools like Vite and Snowpack growing, Meteor is looking horribly outdated (their installation process is interactive even).

**Comment by filipenevola** [comment]
> the deploying sometimes hangs without letting the user know if it's actually frozen or if it's just moving slowly.

what kind of deploy are you talking about @santiagopuentep ? Could you provide a reproduction or at least more details here?

**Comment by santiagopuentep** [comment]
@filipenevola All this in WIndows of course.
